src/praisonai-train/tests/unit/train/test_remote_cli.py

```
@@ -0,0 +1,223 @@
+    import json
+    fake["install"](ok=True)
+    result = runner.invoke(app_mod.app, ["remote", "preflight", "gpubox", "--json"])
+    assert json.loads(result.output)["ok"] is True
+
+
+# --------------------------------------------------------------------------- #
+# start
+# --------------------------------------------------------------------------- #
+def test_start_ships_the_config_and_dataset(fake, tmp_path):
+    r = fake["install"]()
+    cfg = tmp_path / "config.yaml"; cfg.write_text("model_name: x\n")
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)